configure

```diff
@@ -48,6 +48,7 @@ Usage: $0 [OPTION]... [VAR=VALUE]...
     --with-dscat            build with dscat
     --with-example-plugin   build with example plugin
     --with-plugin=DIR       build with plugin in given directory
+    --enable-all-bundled-plugins always enable all bundled plugins
```

Comment:
```suggestion
    --load-bundled-plugins always enable all bundled plugins
```

I find "load" a bit clearer. The "all" quantifier is not adding much IMO.

Reply:
Another option: `--autoload-plugins`

Reply:
We settled on `--with-plugin-autoloading`.